model/likes.js

```diff
@@ -0,0 +1,20 @@
+const likesDatabase = require("./databaseLogic/likesDatabase");
+
+class Like {
+  constructor(likesDatabaseClass = likesDatabase) {
+    this.likesDatabaseClass = likesDatabaseClass;
+  }
```

Comment:
Now I see why you have that weird pattern I guess. Are you doing this for dependency injection, to allow for testing? If that's the case, I would propose instead that you use environment variables, and then check whether the environment variable is set to 'test' or 'production' in your code that connects to the database to choose the right database. That's the typical way.

Reply:
Ah ok I didn't see this comment. Yes, I did this for dependency injection to allow for testing. I'll do some research on implementing a solution that uses environment variables and then try to implement it.